VirtoCommerce.Storefront/VirtoCommerce.Storefront.csproj

```diff
@@ -29,7 +30,8 @@
   <ItemGroup>
     <Compile Remove="wwwroot\cms-content\**" />
     <Content Remove="wwwroot\cms-content\**" />
-    <Content Update="wwwroot\cms-content\**" CopyToPublishDirectory="Never" />
+    <EmbeddedResource Remove="wwwroot\cms-content\**" />
```

Comment:
this line also breaks a publishing process. Storefront mustn't copy wwwroot theme resources in production in default solution.